build.gradle

```diff
@@ -1,19 +1,26 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    ext {
+        kotlin_version = '1.2.71'
```

Comment:
I've found it nice to also have a variable for `support_library_version`

Reply:
Good idea, adding this